simplic-interval/Simplic.Interval.Service/IntervalService.cs

```diff
@@ -0,0 +1,92 @@
+﻿using Dapper;
+using Simplic.Framework.DAL.DatabaseMetadata;
+using Simplic.Sql;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Simplic.Interval.Service
+{
+    public class IntervalService : IIntervalService
+    {
+        #region Fields
+
+        private readonly ISqlService sqlService;
```

Comment:
we use a const string for table names. `private const string TableName = "IT_Interval";`